scripts/deis.sh

```diff
@@ -3,12 +3,24 @@
 deis_healthcheck() {
   wait-for-all-pods "deis"
   local successes
+  local failures
+  local max_attempts=10
   echo "Checking to see if the workflow has come up properly."
-  while [[ ${successes} -lt 10 ]]; do
+  while [[ ${successes} -lt "${max_attempts}" ]] && [[ ${failures} -lt "${max_attempts}" ]]; do
```

Comment:
Do these checks now need to be less than or equal to as well, since the later checks require equal to?